core/src/retransmit_stage.rs

```diff
@@ -352,7 +356,7 @@ pub fn retransmitter(
     bank_forks: Arc<RwLock<BankForks>>,
     leader_schedule_cache: Arc<LeaderScheduleCache>,
     cluster_info: Arc<ClusterInfo>,
-    shreds_receiver: Receiver<Vec<Shred>>,
+    shreds_receiver: Receiver<Vec</*shred:*/ Vec<u8>>>,
```

Comment:
maybe expose `type ShredPayload = Vec<u8>` to use here?

Reply:
Personally not a big fan of typedefs because for someone reading the code, `/*shred:*/ Vec<u8>` is more explicit what type I am working with but for a typedef I have to jump somewhere else in the code to find the definition of typedef and always mentally map the typedef to the actual type when working with that typedef.

I have some further follow up changes to shreds ingestion -> retransmit pipeline path: https://github.com/solana-labs/solana/pull/26300 and https://github.com/solana-labs/solana/pull/26291. Let's revisit this once the code is stable.